src/Microsoft.FeatureManagement.Telemetry.ApplicationInsights/ApplicationInsightsTelemetryPublisher.cs

```diff
@@ -30,17 +30,22 @@ public ValueTask PublishEvent(EvaluationEvent evaluationEvent, CancellationToken
 
             FeatureDefinition featureDefinition = evaluationEvent.FeatureDefinition;
 
-            Dictionary<string, string> properties = new Dictionary<string, string>()
+            var properties = new Dictionary<string, string>()
             {
                 { "FeatureName", featureDefinition.Name },
-                { "IsEnabled", evaluationEvent.IsEnabled.ToString() }
+                { "Enabled", evaluationEvent.Enabled.ToString() }
             };
 
             if (evaluationEvent.Variant != null)
             {
                 properties["Variant"] = evaluationEvent.Variant.Name;
             }
 
+            if (evaluationEvent.VariantAssignmentReason != VariantAssignmentReason.None)
+            {
+                properties["VariantAssignmentReason"] = ToString(evaluationEvent.VariantAssignmentReason);
```

Comment:
Always send Variant if assignment reason is not `None`. Even if variant is null.